src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -75,14 +76,14 @@ public void onDisable() {
 	 * @param path Path to the config var
 	 * @return String
 	 */
-	public String getLang( String path ) {
-		if ( null == this.l10n ) {
+	public String getMessage( String path ) {
+		if ( getL10n() == null ) {

```

Comment:
I realize this is a WordPress thing, but let's stick to using yoda-conditionals here, `null == getL10n()`